### PR TITLE
Add new infoblox module

### PR DESCRIPTION
**Pull request by xoanmi**
##### Issue Type:

Please pick one and delete the rest:
- New Module Pull Request
##### Plugin Name:

infoblox
##### Ansible Version:

```
ansible 1.9.4
```
##### Summary:

Manage Infoblox IPAM and DNS via Web API
##### Example output:

```
(Paste verbatim command output here if necessary)
```

New ansible module to manage infoblox via Web API

### CODE REVIEW

**Comment by xoanmi** [comment]
Hi,

I don't understand why the continuous-integration script complains about the syntax:

```
Compiling ./network/infoblox.py ...
  File "./network/infoblox.py", line 377
    from ansible.module_utils.basic import *
                                            ^
SyntaxError: invalid syntax
```

File: network/infoblox.py [hunk 1]
```diff
@@ -0,0 +1,380 @@
...
+    network     = module.params["network"]
+    address     = module.params["address"]
+    attr_name   = module.params["attr_name"]
+    attr_value  = module.params["attr_value"]
+    comment     = module.params["comment"]
+    api_version = module.params["api_version"]
+    dns_view    = module.params["dns_view"]
+    net_view    = module.params["net_view"]
+
+    try:
```
Comment (sivel) [inline network/infoblox.py:321]: This `try` is unterminated (missing `except`/`finally`).  This is the reason for the syntax errors.

Comment (xoanmi) [inline network/infoblox.py:321]: I've removed the `try` termination dua a previous syntaxys error:

`except Exception as e:     
    module.fail_json(msg=str(e))`

**Comment by xoanmi** [comment]
I've added the except section again and the CI scrip still complains:

```
  File "./network/infoblox.py", line 377
    except Exception as e:
                      ^
SyntaxError: invalid syntax
```

Any idea?

**Comment by sivel** [comment]
`as e` is python 2.6+ syntax, our module guidelines ask that modules support python 2.4 syntax where you would use `, e` instead.

Additionally, your module requires the `requests` library.  Instead of using the requests library, you should use the functionality provided by `ansible.module_utils.urls` to perform HTTP requests.

```
============================================================================
infoblox.py
============================================================================
ERROR: requests import found, should use ansible.module_utils.urls instead
```

**Pull request closed without merging**